project1/applications.py

```diff
@@ -87,3 +87,8 @@ def logout():
     except:
         var = "You must logout from the page"
         return render_template("Registration.html", message1 = var)
+
+@app.route("/bookpage/<id>")
+def bookpage(id):
```

Comment:
This url should not be bypassed. So please check with the session before entering. If by passed return it to login page